Index: src/main/java/LogIn.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package PACKAGE_NAME;\r\n\r\npublic class LogIn {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/LogIn.java	(revision 4cf25c749ce1e33418d2a0a6e3ef7e4d539ff86c)
+++ src/main/java/LogIn.java	(date 1589461737323)
@@ -1,4 +1,59 @@
-package PACKAGE_NAME;
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.*;
+import javafx.stage.Stage;
+
+public class LogIn extends Application {
+
+
+    Button logIn;
+    public static void main(String[] args) {
+
+        launch(args);
+    }
+
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+
+        primaryStage.setTitle("Log In Tab");
+
+        logIn=new Button("Log In");
+
+        GridPane gridLayout=new GridPane();
+        gridLayout.setPadding(new Insets(15, 15, 15, 15));
+        gridLayout.setVgap(7);
+        gridLayout.setHgap(1);
+        gridLayout.setAlignment(Pos.BASELINE_CENTER);
+
+        Label usernameLabel = new Label("Username");
+        Label passwordLabel = new Label("Password");
+        TextField usernameTF = new TextField();
+        TextField passwordTF = new TextField();
+        usernameTF.setPromptText("Create an username");
+        passwordTF.setPromptText("Create a password");
+
+        gridLayout.setConstraints(usernameLabel, 0, 1 );
+        gridLayout.setConstraints(passwordLabel, 0, 2);
+        gridLayout.setConstraints(usernameTF, 1 , 1);
+        gridLayout.setConstraints(passwordTF, 1, 2 );
+        gridLayout.setConstraints(logIn,0,3);
 
-public class LogIn {
+        VBox layout=new VBox(20);
+        gridLayout.getChildren().addAll(usernameLabel,passwordLabel,logIn,usernameTF,passwordTF);
+        layout.setAlignment(Pos.CENTER);
+
+        Scene scene=new Scene(gridLayout,650,250);
+
+        primaryStage.setScene(scene);
+        primaryStage.show();
+
+
+    }
 }
Index: .idea/portal-consultatii.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"JAVA_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\" inherit-compiler-output=\"true\">\r\n    <exclude-output />\r\n    <content url=\"file://$MODULE_DIR$\" />\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/portal-consultatii.iml	(revision 4cf25c749ce1e33418d2a0a6e3ef7e4d539ff86c)
+++ .idea/portal-consultatii.iml	(date 1589461668773)
@@ -1,9 +1,2 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
+<module type="JAVA_MODULE" version="4" />
\ No newline at end of file
